domains/eliasbloem.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "ItzjustElias",
+        "email": "elias.bloem123@gmail.com"
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```

Comment:
I can not reach your website
Netlify says
> Site Not Found
Looks like you've followed a broken link or entered a URL that doesn't exist on Netlify.